pom.xml

```diff
@@ -7,38 +7,39 @@
     <packaging>war</packaging>
 
     <build>
+        <finalName>crh-web</finalName>
```

Comment:
Should we change this to EMT?

Reply:
That would be nice for consistency's sake, but is that gonna be an issue if other parts of the project expect to interact with a .war by the name of crh-web?